src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAddNote.java

```diff
@@ -101,13 +102,16 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
             caseData.setNotes(listValues);
         } else {
+            AtomicInteger listValueIndex = new AtomicInteger(0);
```

Comment:
Seems a bit OTT given this will be on the stack and not concurrent but I guess it doesn't hurt